fdbserver/workloads/FastTriggeredWatches.actor.cpp

```diff
@@ -128,14 +129,17 @@ struct FastTriggeredWatchesWorkload : TestWorkload {
 						// loop
 						if (val == setValue)
 							break;
-						ASSERT(first);
+						ASSERT(!watchCommitted);
+						tr.addWriteConflictRange(singleKeyRange(StringRef()));
```

Comment:
A shorthand for this that may also be a little clearer is:

```suggestion
						tr.addWriteConflictRange(singleKeyRange(""_sr));
```